main.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+let operators = document.querySelectorAll(".operator"),
```

Comment:
this way to declare the variables is possible, but not a good code style.

Reply:
done)